.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+
+repos:
```

Comment:
Might want to set the autoupdate frequency to quarterly.

https://pre-commit.ci/#configuration-autoupdate_schedule

Reply:
i set it to monthly, rare enough i think

Reply:
just curious - why quarterly?  the default weekly seems "good enough" - as it probably won't be updated that often anyway